packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -96,7 +96,12 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                 id: index.toString(),
                 accessor: "item",
                 alignment: column.alignment,
-                Header: typeof column.header === "object" ? column.header.value : column.header,
+                Header:
+                    typeof column.header === "object"
+                        ? column.header.value
+                        : props.preview && (column.header?.trim().length ?? 0) === 0
```

Comment:
1. Was this change from `> 0` to `=== 0` necessary because of the combination with `props.preview`? Asking because there are several other `> 0` checks in your PR and those are fine?
2. You fixed the bug and now there are no snapshot changes, which means all the new behaviour of this PR is not verified in tests. Can you add tests for it? Also for the bug that you fixed?

Reply:
Its because design mode put a nbsp instead, so just checking if the variable is empty is not enough

Reply:
👍  can you address my 2nd point and add some tests for the new stuff? I swear I will approve after that 😛

Reply:
Done